scripts/main/photo.js

```diff
@@ -1021,6 +1021,46 @@ photo.getArchive = function (photoIDs, kind = null) {
 	location.href = "api/Photo::getArchive" + lychee.html`?photoIDs=${photoIDs.join()}&kind=${kind}`;
 };
 
+photo.qrCode = function (photoIDs) {
+	if (photoIDs.length === 1) {
+		let myPhoto;
+
+		if (photo.json && photo.json.id === photoIDs[0]) {
+			myPhoto = photo.json;
+		} else {
+			myPhoto = album.getByID(photoIDs[0]);
```

Comment:
If you only use the first element from the array, why do you pass an array at all? This is confusing.

Reply:
https://github.com/LycheeOrg/Lychee-front/pull/287/commits/55fe0cb2375ba5d9c5b25f5db178c189a4e5b6bc